jsconfig.json

```diff
@@ -1,10 +1,9 @@
-
 {
-	"compilerOptions": {
-		"module": "ES6",
-		"jsx": "preserve",
-		"checkJs": true,
-		"allowUnreachableCode": true
-	},
-	"exclude": [ "node_modules" ]
+  "compilerOptions": {
```

Comment:
is this file still relevant?

Reply:
True! This is a stand-in for `tsconfig` [for VSCode](https://code.visualstudio.com/docs/languages/jsconfig), but now we have a legitimate TypeScript configuration.

Reply:
It's def not relevant still, can be killed off

Reply:
Done